Make the readme more streamline

fgrunewald opened the pull request:
This new version of the Readme draws more attention to the wiki and migrates all essential information there. This keeps the front page more clean and allows us to fix simple problems in the tutorial/installation instructions without having to edit the Readme every time.

@ricalessandri and @pckroon could you give some general comments feedback?

Also together with this PR the following wiki pages have been created. Ideally someone check those as well. You should be able to edit and modify them as you see fit. Let me know if you have any issues:

https://github.com/marrink-lab/polyply_1.0/wiki/Quick-Start
https://github.com/marrink-lab/polyply_1.0/wiki/Installation
https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-parameters

pckroon commented on README.md:
```diff
@@ -3,82 +3,21 @@
-some commonly used macromolecules using different force-fields (martini, gromos) is included.
-In principle the program can be used with any type of force-field (FF).
+Polyply is a python suite designed to facilitate the generation of input files and coordinates for simulating
+(bio)macromolecules such as synthetic polymers or polysacharides. Input files can be generated either from user specified building blocks or by using the polymers availbable in the library. The library currently includes polymer definitions for the GROMOS and Martini force-fields. The [quick start](https://github.com/marrink-lab/polyply_1.0/wiki/Quick-Start) section in the wiki gives an overview  ...
```
Break over multiple lines, contains at least one typo

ricalessandri replied:
typos: polysacharides, availbable

pckroon commented on README.md:
```diff
@@ -3,82 +3,21 @@
-chirality is also not taken into account. All polymers are atactic unless a dihedral specifies the 
-chirality.
+## Quick references
+[Installation Guide](https://github.com/marrink-lab/polyply_1.0/wiki/Installation)<br/>
+[FAQs](https://github.com/marrink-lab/polyply_1.0/wiki/FAQs)<br/>
+[Submissions to martini polymer library](https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-parameters)<br/>
+[Tutorial Martini Polymers](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-martini-polymer-melts)<br/>
+[Tutorial GROMOS Polymers](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-GROMOS-polymer-melts)<br/>
```
The `<br\>` line breaks look weird in markdown. Change them to `\` (https://stackoverflow.com/questions/26626256/how-to-insert-a-line-break-br-in-markdown)

ricalessandri replied:
I like the quick references. 

Small thingy: you are capitalizing "Installation Guide" and "Tutorial Martini Polymers" but not "Submissions to martini polymer library" - I would be consistent.

pckroon commented on README.md:
```diff
@@ -3,82 +3,21 @@
-are welcome as [bug reports] and [pull requests]. Note however that the
-decision of whether or not contributions can give authorship on the resulting
-academic paper is left to our sole discretion.
+We are happy to accept submissions of polymer parameters (mostly for the Martini force-field). After a small quality control procedure, parameters are distributed via the Martini library including appropiate citations. To submit parameters simply open an [issue][bug reports]. More details on submitting parameters can be found [here](https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-p ...
```
Break over multiple lines

ricalessandri replied:
for the rest looks good to me

pckroon reviewed (changes requested):
Couple of nitpicks on the markdown formatting, but the text looks good to me.

ricalessandri reviewed (changes requested):
Few things noted but overall looks nice

ricalessandri commented:
I did make also some (cosmetic) changes to:
- https://github.com/marrink-lab/polyply_1.0/wiki/Quick-Start
- https://github.com/marrink-lab/polyply_1.0/wiki/Installation
- https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-parameters

fgrunewald commented:
@pckroon and @ricalessandri thanks for the quick feedback! I added line breaks and fixed the typos as well as made the quick-reference section consistent. I also added the citation to the archive paper. 

@ricalessandri thanks for the fixes in the wiki sections.

pckroon reviewed (approved):
LGTM!
